### ISSUE TITLE
conflicting dependencies in v2.0.0-alpha7

### ISSUE THREAD

**Issue by carocad** [metosin/compojure-api#333]
Running `lein deps :tree` produces:
```clojure
Possibly confusing dependencies found:
[metosin/compojure-api "2.0.0-alpha7"] -> [ring/ring-core "1.6.2"] -> [clj-time "0.11.0"]
 overrides
[metosin/compojure-api "2.0.0-alpha7"] -> [metosin/ring-swagger "0.24.1"] -> [clj-time "0.14.0"]

Consider using these exclusions:
[metosin/compojure-api "2.0.0-alpha7" :exclusions [clj-time]]

[metosin/compojure-api "2.0.0-alpha7"] -> [ring/ring-core "1.6.2"] -> [ring/ring-codec "1.0.1"] -> [commons-codec "1.6"]
 overrides
[metosin/compojure-api "2.0.0-alpha7"] -> [metosin/muuntaja "0.3.2"] -> [com.cognitect/transit-clj "0.8.300"] -> [com.cognitect/transit-java "0.8.324"] -> [commons-codec "1.10"]

Consider using these exclusions:
[metosin/compojure-api "2.0.0-alpha7" :exclusions [commons-codec]]

[metosin/compojure-api "2.0.0-alpha7"] -> [ring/ring-core "1.6.2"] -> [clj-time "0.11.0"] -> [joda-time "2.8.2"]
 overrides
[metosin/compojure-api "2.0.0-alpha7"] -> [metosin/ring-swagger "0.24.1"] -> [clj-time "0.14.0"] -> [joda-time "2.9.7"]

Consider using these exclusions:
[metosin/compojure-api "2.0.0-alpha7" :exclusions [joda-time]]
```

The problem is that by following the advice from lein, the project is not compilable anymore since the exclusions are actually needed.

**Comment by ikitommi** [comment]
Would you like to do a PR?

**Comment by carocad** [comment]
@ikitommi happy to do it. To which repo should I do it though? there are lots of `metosin` repos in the chain of conflict :/

**Comment by ikitommi** [comment]
Please fix 'em all :)

**Comment by ikitommi** [comment]
Fixed in `2.0.0-alpha17`
